deploy/registration-service/registration-service.yaml

```diff
@@ -31,13 +31,6 @@ objects:
       - masteruserrecords
       verbs:
       - get
-    - apiGroups:
-      - toolchain.dev.openshift.com
-      resources:
-      - bannedusers
-      verbs:
-      - get
-      - list
```

Comment:
Please, revert these changes - it's because of https://github.com/codeready-toolchain/host-operator/pull/151#issuecomment-594006688

Reply:
done in 88408c2